apis/v1alpha1/backendpolicy_types.go

```diff
@@ -97,16 +97,17 @@ type BackendRef struct {
 
 // BackendTLSConfig describes TLS configuration for a backend.
 type BackendTLSConfig struct {
-	// CertificateAuthorityRef is a reference to a resource that includes
-	// trusted CA certificates for the associated backends. If an entry in this
-	// list omits or specifies the empty string for both the group and the
-	// resource, the resource defaults to "secrets". An implementation may
-	// support other resources (for example, resource "mycertificates" in group
-	// "networking.acme.io").
+	// CertificateAuthorityRef is a reference to a Kubernetes object that contains
+	// one or more trusted CA certificates. The CA certificates are used to establish
+	// a TLS handshake to backends listed in BackendRefs. The referenced object MUST
+	// reside in the same namespace as BackendPolicy.
 	//
-	// When stored in a Secret, certificates must be PEM encoded and specified
-	// within the "ca.crt" data field of the Secret. Multiple certificates can
-	// be specified, concatenated by new lines.
+	// CertificateAuthorityRef can reference a standard Kubernetes resource, i.e.
+	// ConfigMap, or an implementation-specific custom resource.
+	//
+	// When stored in a Secret, certificates must be PEM encoded and specified within
+	// the "ca.crt" data field of the Secret. When multiple certificates are specified,
+	// the certificates MUST be concatenated by new lines.
```

Comment:
```suggestion
	// When stored in a Secret, certificates must be PEM encoded and specified within
	// the "ca.crt" data field of the Secret. When multiple certificates are specified,
	// the certificates MUST be concatenated by new lines.
	//
	// CertificateAuthorityRef can also reference a standard Kubernetes resource, i.e.
	// ConfigMap, or an implementation-specific custom resource.
```

Starting off with Secret resource, which has more clear requirements makes it a little easier to understand.

Reply:
+1